roles/besu/tasks/setup.yaml

```diff
@@ -28,3 +28,4 @@
     networks: "{{ besu_container_networks }}"
     command: "{{ besu_container_command + besu_container_command_extra_args }}"
     user: "{{ besu_user_meta.uid }}"
+    pull: "{{ image_pull_policy }}"
```

Comment:
Please follow the same naming conventions that have been used. 

In this case it should be renamed from `image_pull_policy` to `besu_container_pull`. 

It's important to have different names for each role since ansible vars have a global scope.

Reply:
The same logic applies to the other roles.